ihmc-footstep-planning/src/main/java/us/ihmc/footstepPlanning/log/FootstepPlannerLogger.java

```diff
@@ -270,6 +291,7 @@ public boolean logSessionWithExactFolderName(String sessionDirectory)
          }
 
          fileWriter.flush();
+         fileWriter.close();
```

Comment:
This was a bug on windows, so if you tried to delete the log you just created it wouldn't work because these files weren't getting closed. For some reason Ubuntu could figure it out but on Windows these files would be left cause one of the `FootstepPlannerLoggerTest` to fail